docs/guide/admin_panel.md

```diff
@@ -263,3 +263,18 @@ By default, eZ Platform contains one Object state group: **Lock**, with states *
 
 Object states can be used in conjunction with permissions, in particular with the [State Limitation](limitation_reference.md#state-limitation).
 Their specific use cases depend on your needs and the setup of your permission system.
+
+## Segments
+
+Segments are used to display specific content to specific Users.
```

Comment:
We might consider going for the active voice here:
"You use segments to display ..."